README.md

```diff
@@ -1,2 +1,95 @@
-# ES6_Awesome_Books
-Creation Book Library with ES6 Features
+# Awesome_Books
+Creation Book List Library for a user to store the list of books 
+
+> A demo website for Book Library.
+
+## Built With
+
+HTML and CSS
+Javascript : Arrays and Objects , DOM , Local Storage etc 
+
+## Video
+
+In the following [link](Coming Soon)
+
+
+## Live Server
+
+In the following link, you can view (Coming Soon)).
+
+## Screenshot
+
+![Book_Store_App(Page1)](./images/Image1.png)
+![Book_Store_App(Page2)](./images/Image2.png)
+![Book_Store_App(Page3)](./images/Image3.png)
+
+### Getting Started
+
+To get a local copy follow these steps:
+
+1. Clone the repo by typing on your terminal
+
+```
+git clone https://github.com/RaviTejaM9602/ES6_Awesome_Books
+```
+
+2. Access the repo with
+
+```
+cd Conference
+```
+
+3. Clone the Repo or Download the Zip file
+Run the live server or simply open index.html with your browser.
+
+```
+code .
+```
+
```

Comment:
- [ ] As the previous CR already mentioned please add instructions on how to setup your project right below this section like the command `npm install`, if the user doesn't run the `npm install` the application doesn't work because the application will return an error due to Luxon not being installed.